### ISSUE TITLE
Doodle Classifier: Getting error "No member named 'setTo' in 'ofXml'" in ofApp.cpp

### ISSUE THREAD

**Issue by Martinukai** [ml4a/ml4a-ofx#30]
I am getting these various errors in the ofApp.cpp file when compiling:

line 43: No member named 'setTo' in 'ofXml'
line 44 -> 46: No matching member function for call to 'getValue' 
line 47: No member named 'exists' in 'ofXml'
line 48: No member named 'setTo' in 'ofXml'
line 54: No member named 'setToSibling' in 'ofXml'

Any tips would be much appreciated. Thanks.

**Comment by genekogan** [comment]
the api of ofXml has changed since 0.10.0 release, so we need to update it. for now you can get it working again using an older openframeworks (0.9.*) until we fix it. if you manage to, a PR would be very welcome.

**Comment by Monoth** [comment]
Thanks for the speedy response. What is a PR? I'm not an advanced coder ;)

**Comment by heaversm** [comment]
This applies to a number of things I would imagine. I came across the same error on KeyboardOSC in the osc-modules directory. I would be happy to do a PR, but it's not immediately apparent how things should map. For example, @genekogan , in .9.x we have `bool ofXml::setTo(const string& path){` , and in `.10.x` it seems we only have `void ofXml::setName(const std::string & name){`.

One is a bool, one is void, one is looking or a path, one is looking for a name, etc. If you have any pointers on where to get started, I could look into it.

**Comment by genekogan** [comment]
yes, all the xml needs to be redone either using the native ofXml or ofJson objects (not ofxXml or ofxJson) or using ofxXmlSettings. if anyone wants to take a shot and submit a pull request, please do! otherwise i should get to it sometime next few weeks...

**Comment by PrinceSajjadHussain** [comment]
[Uploading Lecture 22.pptx…]()
